x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -2353,6 +2723,51 @@ export class RulesClient {
     }
     return alertAttributes;
   }
+
+  private applyBulkEditOperation<R extends object>(operation: BulkEditOperation, rule: R) {
```

Comment:
**nit:** could this be moved to a separate file?

Reply:
moved to a separate file. Thanks for suggestion